### ISSUE TITLE
Gist settings stopped syncing

### ISSUE THREAD

**Issue by haoss** [shanalikhan/code-settings-sync#1394]
There are 2 working computers, on the first, working, Ubuntu 20.04 I save the settings in the gist https://gist.github.com/haoss/d360e3dfd8a0934325f5115e49c3a920 on the second, when synchronizing, we get this https://i.imgur.com/121Vmiv.png and nothing is synced.

**Comment by PabloPenia** [comment]
Same issue here, I have installed on a laptop  with manjaro, just  get a new computer tried to sync (same OS) but nothing is installed (no errors)

**Comment by haoss** [comment]
> Same issue here, I have installed on a laptop with manjaro, just get a new computer tried to sync (same OS) but nothing is installed (no errors)

until the problem is solved, I used the built-in function via github at this [link](https://code.visualstudio.com/docs/editor/settings-sync), checked, everything works

**Comment by PabloPenia** [comment]
yeah but that option doesn't exist on the community version (vscodium).

**Comment by haoss** [comment]
> yeah but that option doesn't exist on the community version (vscodium).

I use the free version for Windows and Linux, I don't know about your version
